### PR TITLE
Running Midship code

### PR DESCRIPTION
**Pull request by sohil1234**
- [ ] Code has been scanned with [SonarLint](http://www.sonarlint.org/intellij/)
- [ ] Methods have been annotated with @ Nullable and @ Nonnull ([Read more](https://github.com/MovingBlocks/DestinationSol/wiki/@Nullable-&-@Nonnull-Quickstart))
- [ ] There are no errors present in the project
- [ ] Code has been formatted and indented
- [ ] Methods have appropriate Javadoc ([How to write Javadoc](http://www.oracle.com/technetwork/java/javase/documentation/index-137868.html))

### CODE REVIEW

**Comment by GooeyHub** [comment]
Can one of the admins please verify this patch?

**Comment by vampcat** [comment]
Okay, so this PR does not contain extra changed files, which is definitely good. However, I would like you to create a single PR for all the 5 parts of the task. That means you can just keep adding commits to this branch, and they will show up on this PR.


In order to do that, you need to copy over the files for your previous vulcan ship onto this PR. Note that wile doing that, the icon for vulcan ship should be changed to pure white instead of grayscale (look at other icons for reference). Further, the icon for this ship does not match the ship, and neither is it white. You'll have to fix that as well.

**Comment by vampcat** [comment]
This PR is almost good to go, apart from the icons.

As I mentioned before, we use *white* icons. Your icons are still grayscale, which is not the same as white. Also, your starTrek icon is not the same as ship, so you should probably fix that as well.

In short, fix the icons, and we're good to go.

**Comment by vampcat** [comment]
Also, can you please make the PR description a bit more.. reviewer-friendly? You are supposed to explain to someone who's never seen this PR before *what* this PR is about, what it adds and how they can test it - have a look at any of the other PRs for reference.

File: modules/core/assets/items/clips/railClip/railClip.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
     "projectile": "rail",
-    "size": 5,
-    "price": 70,
+    "size": 200,
+    "price": 20,
```
Comment (vampcat) [inline modules/core/assets/items/clips/railClip/railClip.json]: As I mention before, I see no reason to change these guns in core.

Is there any particular reason for doing this, that I'm missing?

File: modules/core/assets/items/clips/slugClip/slugClip.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
     "projectile": "slug",
-    "size": 10,
-    "price": 60,
+    "size": 100,
```
Comment (vampcat) [inline modules/core/assets/items/clips/slugClip/slugClip.json]: Same comment as above - why are you changing thing?

**Comment by vampcat** [comment]
@GooeyHu: ok to test

**Comment by GooeyHub** [comment]
Hooray Jenkins reported success with all tests good!

**Comment by GooeyHub** [comment]
Hooray Jenkins reported success with all tests good!

**Comment by GooeyHub** [comment]
Hooray Jenkins reported success with all tests good!

File: modules/vulcan/assets/configs/startingStation.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
-  "hull": "core:Station",
-  "items": " ",
+  "hull": "vulcan:starTrekStation",
+  "items": "vulcan:electric-1 vulcan:electric-2 ",
```
Comment (vampcat) [inline modules/vulcan/assets/configs/startingStation.json]: Remove the extra space at the end of the string.

File: modules/vulcan/assets/configs/startingStation.json [hunk 4]
```diff
@@ -1,6 +1,6 @@
...
   "guard": {
     "hull": "vulcan:starTrekSmall",
```
Comment (vampcat) [inline modules/vulcan/assets/configs/startingStation.json]: Change the guards to something more substantial. Capital ships?

**Comment by GooeyHub** [comment]
Hooray Jenkins reported success with all tests good!

**Comment by GooeyHub** [comment]
Hooray Jenkins reported success with all tests good!

**Comment by GooeyHub** [comment]
Hooray Jenkins reported success with all tests good!

**Comment by GooeyHub** [comment]
Hooray Jenkins reported success with all tests good!

**Pull request merged**